primap2/tests/data/BURDI_conversion.csv

```diff
@@ -0,0 +1,40 @@
+# references: non_annex1_data repo
+# last_update: 2024-10-14
+BURDI,IPCC2006_PRIMAP,comment
+1,1
+1.A,1.A
+1.A.1,1.A.1
+1.A.2,1.A.2
+1.A.3,1.A.3
+1.A.4,1.A.4
+1.A.5,1.A.5
+1.B,1.B
+1.B.1,1.B.1
+1.B.2,1.B.2
+2 + 3,2
+2.A,2.A
+2.B + 2.E,2.B
+2.C,2.C
+2.F,2.F
+2.G + 2.D, 2.H
+3,2.D
+4,M.AG
+4.A,3.A.1
+4.B,3.A.2
+4.C,3.C.7
+4.D + 4.C + 4.E + 4.F + 4.G,3.C
```

Comment:
It would be good to also map 4.D to single category, else we have some of the 3.C subcategories but not all which can lead to confusion and problems in category aggregation and consistency checks. I would add an M-category to IPCC2006_PRIMAP